assets/scss/components/elements/navigation/_nav-styles.scss

```diff
@@ -56,18 +56,25 @@
 		}
 	}
 
-	.wrap:hover > a {
-		color: currentColor;
-
-		&,
-		+ .caret-wrap,
-		span,
-		i {
-			color: var(--hovertextcolor, var(--color)) !important;
+	.wrap:hover {
+
+		svg {
+			color: var(--hovertextcolor, var(--color));
 		}
 
-		&::after {
-			width: calc(100% + var(--spacing));
+		> a {
+			color: currentColor;
+
+			&,
+			+ .caret-wrap,
+			span,
+			i {
+				color: var(--hovertextcolor, var(--color)) !important;
+			}
+
+			&::after {
+				width: calc(100% + var(--spacing));
+			}
```

Comment:
Just a question here, does the code here also work as expected with the mega menu?